pages/index.tsx

```diff
@@ -1,10 +1,31 @@
 import React, { memo } from "react";
+import PropTypes from "prop-types";
 import { compose } from "redux";
 import { injectIntl } from "react-intl";
 import { TracksContainer } from "@features/itunes";
+import { getRecommendations } from "../services/root";
 
-export const TracksPage = () => {
-  return <TracksContainer />;
+export async function getStaticProps() {
+  const recommendations = await getRecommendations();
+
+  return {
+    props: {
+      recommendations,
+    },
+  };
+}
```

Comment:
Although it's not a hard requirement, prefer keeping the data fetching at the very bottom of the file.

Reply:
Got it.